### PR TITLE
Added sqlite3

### PR DESCRIPTION
**Pull request by meguminloli**
Create the database using the createdatabase.sh . On Windows, just copy the command. It will generate a file called `message.db`, the sqlite3 database.

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import requests
 import json
+import sqlite3
 
-token = ""
+token = "1794848670:AAEzdrn_7BzpbWEPBX8rcCHvPvuH8q44Yfw"
```
Comment (hydrastro) [inline bot.py:5]: This token should be replaced with a placeholder: token = "placeholder"

File: bot.py [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 buffer_size = 1000
```
Comment (hydrastro) [inline bot.py:6]: with sqlite support, this variable is useless, and should be removed

File: bot.py [hunk 3]
```diff
@@ -18,39 +20,42 @@ def send_message(message, chat_id):
...
-def get_reply(database, message_info):
+def get_reply(message_info):
     reply = None
     message = message_info["message"]
+    print(message)
     if message is not None:
         if message.startswith("s/"):
             if message.endswith("/"):
                 message = message[:-1]
+            elif message.endswith("g"):
```
Comment (hydrastro) [inline bot.py:42]: this statement needs to be removed (is it a debugging leftover?)

File: test.sql [hunk 4]
```diff
@@ -0,0 +1 @@
+SELECT * FROM messages;
```
Comment (hydrastro) [inline test.sql:1]: this file can be removed

File: createdatabase.sh [hunk 5]
```diff
@@ -0,0 +1 @@
+cat createdb.sql | sqlite3 messages.db
```
Comment (hydrastro) [inline createdatabase.sh:1]: We could put this in `README.md`, what do you think?
Also I think we should remove `messages.db`

**Review by hydrastro — CHANGES_REQUESTED** [review]
Good work!
We're close to merging,
Just tell me if I should merge this now and change these things later or if you want to do it instead (it would be neater)

File: bot.py (hunk 3 above)
Comment (meguminloli) [reply bot.py:42]: no, it's the global selector in sed, maybe someone is a tryhard

File: createdatabase.sh (hunk 5 above)
Comment (meguminloli) [reply createdatabase.sh:1]: ok

**Comment by meguminloli** [comment]
Now, it should be done.

File: bot.py (hunk 3 above)
Comment (hydrastro) [reply bot.py:41]: oh, got it. Nice!

**Review by hydrastro — APPROVED** [review]

**Pull request merged**